tox.ini

```diff
@@ -48,7 +48,7 @@ commands =
 
 [flake8]
 show-source = True
-ignore = D100,D101,D102,D103,D104,D105,D107,A002,A003,W504
+ignore = D100,D101,D102,D103,D104,D105,D107,A002,A003,A005,W504
```

Comment:
sounds unrelated?

Reply:
That was required to get the `pep8` check to pass. Fixing `A005` properly would require renaming functions so they aren't shadowing built-ins, and I didn't want to introduce any more functional changes than I had to, so I added it to the ignore list.

Reply:
for reference:
> ./gnocchiclient/v1/resource.py:0:1: A005 the module is shadowing a Python builtin module "resource"